index.html

```diff
@@ -42,26 +49,27 @@ <h1 id="myName">
                             <a href=""><img src="img/github.svg" alt="GitHub" /></a>
                         </li>
                         <li class="icons">
-                            <a href=""><img src="img/Linkedin icon.svg" alt="GitHub" /></a>
+                            <a href=""><img src="img/Linkedin icon.svg" alt="LinkedIn" /></a>
                         </li>
                         <li class="icons">
-                            <a href=""><img src="img/angel list.svg" alt="GitHub" /></a>
+                            <a href=""><img src="img/angel list.svg" alt="angel" /></a>
                         </li>
                         <li class="icons">
-                            <a href=""><img src="img/twitter.svg" alt="GitHub" /></a>
+                            <a href=""><img src="img/twitter.svg" alt="twitter" /></a>
                         </li>
                         <li class="icons">
-                            <a href=""><img src="img/medium.svg" alt="GitHub" /></a>
+                            <a href=""><img src="img/medium.svg" alt="Medium" /></a>
                         </li>
                     </ul>
                 </div>
             </div>
         </section>
-       <section class="works">
+       <section id="projects" class="works">
         <div class="recent-works">
             <h2 id="works-title">My Recent Works</h2>
             <hr class="works-break">
         </div>
+        
         <div id="grid-container">
             <div class="recent-project">
```

Comment:
- [ ] As mentioned in the previous review, let us `blur` the background to match the original design, also let us align the buttons to match the text as seen in the original design. Image is broken, let us fix it and increase the width to match
![Screenshot from 2023-05-27 01-10-38](https://github.com/Obimbo07/Portfolio/assets/46388113/8ff94a44-d8a4-4313-af9e-2f4daa0d14b2)

Reply:
[OPTIONAL ] 
- Great job making the background blur and aligning the button as requested by previous reviewers. I am making this change optional exceptionally and I strong advise you effect the change. **Image is broken, let us fix it and increase the width to match**
![image](https://github.com/Obimbo07/Portfolio/assets/62849872/34d358dd-baea-4928-adbc-529fcf21d700)